io.github.cortex_command_community.Cortex-Command-Community-Project.yml

```diff
@@ -0,0 +1,67 @@
+app-id: io.github.cortex_command_community.Cortex-Command-Community-Project
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: CortexCommand
+rename-desktop-file: CortexCommand.desktop
+rename-icon: io.github.cortex_command_community.Cortex-Command-Community-Project
+finish-args:
+  - --device=dri
+#  - --device=input
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --socket=pulseaudio
+  - --share=network
+cleanup:
+  - '/include'
+  - '/lib/pkgconfig'
+  - '/lib/cmake'
+  - '/lib/*.a'
+modules:
+  - shared-modules/luajit/luajit.json
+  - name: minizip
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DMZ_COMPAT=ON
+      - -DBUILD_SHARED_LIBS=ON
+    cleanup: 
+      - '/share'
+    sources:
+      - type: git
+        url: https://github.com/zlib-ng/minizip-ng.git
+        commit: 4.0.7
+  - name: oneTBB
+    buildsystem: cmake-ninja
+    cleanup: 
+      - '/share'
+    config-opts:
+      - -DTBB_STRICT=OFF
+      - -DTBB_TEST=OFF
+    sources:
+      - type: git
+        url: https://github.com/oneapi-src/oneTBB.git
+        commit: v2021.13.0
+  - name: cortexcommand
+    buildsystem: meson
+    config-opts:
+      - -Duse_pch=enabled
+      - -Dinstall_fmod=true
+      - -Dinstall_data=true
+      - -Dinstall_runner=true
+    sources:
+      - type: git
+        url: https://github.com/cortex-command-community/Cortex-Command-Community-Project
+        commit: v6.2.2
+      - type: patch
+        path: runner.patch
+  - name: ccicon
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 io.github.cortex_command_community.Cortex-Command-Community-Project.svg -t ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/
+      - install -Dm644 io.github.cortex_command_community.Cortex-Command-Community-Project.metainfo.xml -t ${FLATPAK_DEST}/share/metainfo/
+    sources:
+      - type: file
+        path: io.github.cortex_command_community.Cortex-Command-Community-Project.svg
+      - type: file
+        path: io.github.cortex_command_community.Cortex-Command-Community-Project.metainfo.xml
```

Comment:
merge the sources with the cortexcommand module and add the build-commnads as `post-install`.